labs/unicorn-store/software/alternatives/unicorn-store-micronaut/pom.xml

```diff
@@ -4,57 +4,75 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.unicorn.store</groupId>
   <artifactId>store-micronaut</artifactId>
-  <version>1.0.0</version>
+  <version>2.0.0</version>
   <packaging>${packaging}</packaging>
 
   <parent>
     <groupId>io.micronaut.platform</groupId>
     <artifactId>micronaut-parent</artifactId>
-    <version>4.1.5</version>
+    <version>4.1.6</version>
   </parent>
   <properties>
     <packaging>jar</packaging>
     <jdk.version>17</jdk.version>
     <release.version>17</release.version>
+    <micronaut.version>4.1.6</micronaut.version>
+    <micronaut.aot.enabled>false</micronaut.aot.enabled>
```

Comment:
Why do we provide AOT configuration when we disable it here?

Reply:
I noticed it as well. That's the way Micronaut generates it for new projects. Shall we remove it to clean it up?

Reply:
I assume this is because it is still marked as experimental - however, I think we should enable it as it gives as an additional performance boost.

Reply:
Ok enabled it.